src/test/java/org/example/DeliverToTest.java

```diff
@@ -0,0 +1,100 @@
+package org.example;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class DeliverToTest {
+
+    @Test
+    public void deliverFromAmazon() throws InterruptedException {
+
+        System.setProperty(
+                "webdriver.chrome.driver",
+                "C:\\Users\\hends\\IdeaProjects\\Group4\\src\\test\\resources\\webdriver\\chromedriver.exe");
+
+        WebDriver driver = new ChromeDriver();
+
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+
+        driver.get("https://www.amazon.com/");
+
+        driver.findElement((By.id("glow-ingress-line2"))).click();
+        driver.findElement(By.id("GLUXZipUpdateInput")).sendKeys("99801" + Keys.ENTER);
+        driver.findElement((By.cssSelector(".a-popover-footer #GLUXConfirmClose"))).click();
+
+        Thread.sleep(1000);
+
+        Assert.assertTrue(driver.findElement((By.id("glow-ingress-line2"))).getText().contains("Juneau 99801"));
+
+        driver.close();
+    }
+
+    @Test
+    public void deliverFromCountries() throws InterruptedException{
+
+        System.setProperty(
+                "webdriver.chrome.driver",
+                "C:\\Users\\hends\\IdeaProjects\\Group4\\src\\test\\resources\\webdriver\\chromedriver.exe");
+
+        WebDriver driver = new ChromeDriver();
+
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
```

Comment:
The same code was repeated three times. Consider extracting it to a separate method using `@BeforeTest` annotation.